.github/workflows/chroma-test.yml

```diff
@@ -12,10 +12,17 @@ on:
 
 jobs:
   test:
+    timeout-minutes: 90
     strategy:
       matrix:
         python: ['3.10']
         platform: [ubuntu-latest]
+        testfile: ["--ignore-glob chromadb/test/property/*",
```

Comment:
Just leaving as a note - we will have to remember to add new property test files here.